metadata-utils/src/main/java/com/linkedin/metadata/restli/DefaultRestliClientFactory.java

```diff
@@ -41,56 +42,47 @@ public static RestClient getRestLiD2Client(@Nonnull String restLiClientD2ZkHost,
   }
 
   @Nonnull
-  public static RestClient getRestLiClient(@Nonnull String restLiServerHost, int restLiServerPort) {
-    return getRestLiClient(restLiServerHost, restLiServerPort, false, null);
+  public static RestClient getRestLiClient(@Nonnull String restLiServerHost, int restLiServerPort, boolean useSSL,
```

Comment:
Just to clarify my understanding - this is just simplifying this interface to require less explicit inputs to derive the client?

Reply:
I want to be able to set the context path. I am mounting the restli servlet at /gms on the existing mce-consumer service. Its more flexible to be able to move different services around. Certainly it was more common practice before microservices, hosting multiple services/servlets, than it is today however I think its useful as it could take advantage of certain reverse proxies that handle connections better than k8 services.